src/vkbd/usb-gadget.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+modprobe libcomposite
+cd /sys/kernel/config/usb_gadget
```

Comment:
I would make this `cd /sys/kernel/config/usb_gadget || exit` just in case.

Reply:
Yes.